### PR TITLE
Implemented DDF for cloud subnets and added support mixin call

### PR DESCRIPTION
**Pull request by skateman**
Screenshot from 2021-02-16 12-08-59

Parent issue: https://github.com/ManageIQ/manageiq-ui-classic/issues/6861
@miq-bot add_label enhancement

### CODE REVIEW

File: app/models/manageiq/providers/ibm_cloud/power_virtual_servers/network_manager/cloud_subnet.rb [hunk 1]
```diff
@@ -5,6 +5,49 @@ class ManageIQ::Providers::IbmCloud::PowerVirtualServers::NetworkManager::CloudS
...
 
+  def self.params_for_create(_ems)
+    {
+      :fields => [
+        {
+          :component => 'select',
+          :name      => 'type',
+          :id        => 'type',
+          :label     => _('Type'),
+          :options   => [
```
Comment (skateman) [inline app/models/manageiq/providers/ibm_cloud/power_virtual_servers/network_manager/cloud_subnet.rb:16]: @jaywcarman is this field a two-state, or can it be also empty?

Comment (jaywcarman) [reply app/models/manageiq/providers/ibm_cloud/power_virtual_servers/network_manager/cloud_subnet.rb:16]: This field must be one of the two states.

Comment (skateman) [reply app/models/manageiq/providers/ibm_cloud/power_virtual_servers/network_manager/cloud_subnet.rb:16]: Okay, in that case it should be required...changing

**Review by jaywcarman — APPROVED** [review]
LGTM. @agrare since this form update won't be included in Lasker should we wait to merge? I don't see any obvious harm, but can understand general prudence.

**Comment by skateman** [comment]
> LGTM. @agrare since this form update won't be included in Lasker should we wait to merge? I don't see any obvious harm, but can understand general prudence.

@jaywcarman the change is harmless as long as the UI is not consuming the form schema...

**Comment by Fryguy** [comment]
Skipping backport to `morphy`, because it is already in the branch.

**Pull request merged**